pkg/logs/input/docker/ad_identifier_kubelet.go

```diff
@@ -37,7 +37,7 @@ func ContainsADIdentifier(c *Container) bool {
 	if err != nil {
 		return false
 	}
-	for _, container := range pod.Status.Containers {
+	for _, container := range append(pod.Status.InitContainers, pod.Status.Containers...) {
```

Comment:
nits: we could create a method to return `allContainers` or even a computed property so that we don't have to recreate this array all the time we need it